File: Behavioral_Design_Pattern/Observer Design Pattern/WithJava/WithoutObserverPattern/WeatherApp.java
package WithoutObserverPattern;

public class WeatherApp {
    public static void main(String[] args) {
        PhoneDisplay phoneDisplay = new PhoneDisplay();
        TVDisplay tvDisplay = new TVDisplay();
        WeatherStation weatherStation = new WeatherStation(phoneDisplay, tvDisplay);
        
        weatherStation.setWeather("Sunny");
    }
}

File: System Design Python Codes/Behavioral Design Pattern/Observer_Design_Pattern/Without/WeatherApp.py
from WeatherStation import WeatherStation
from PhoneDisplay import PhoneDisplay
from TVDisplay import TVDisplay

if __name__ == "__main__":
    phoneDisplay = PhoneDisplay()
    tvDisplay = TVDisplay()
    
    weatherStation = WeatherStation(phoneDisplay, tvDisplay)
    
    weatherStation.setWeather("Holocasty")
    
    